sources/Common/include/Constants.h

```diff
@@ -28,6 +28,7 @@ namespace constants {
 const std::string diagramDirectorySuffix{"_Diagram"};              ///< Suffix for the diagram directory
 const std::string svcModelName{"SecondaryVoltageControlSimp"};     ///< name of the Secondary Voltage Controller model
 const std::string rpcl2PropertyName{"ReactivePowerControlLoop2"};  ///< name of the property to annote RPCL2 generators in assembling
+const std::string svcToUMeasurement{"SVCToUMeasurement"};          ///< name of the UMeasurement
```

Comment:
This can be named as we want (it could be called TOTO) so we need another way to detect this is the node connection